app/views/home/index.html.erb

```diff
@@ -12,10 +12,12 @@
   <% end %>
 </div>
 <div class="home__cta-wrap">
-  <h2 class="home__downloads">
-    <p><%= number_with_delimiter(@downloads_count) %></p>
-    <span class="home__downloads__desc">downloads &amp; counting</span>
-  </h2>
+  <% if @downloads_count %>
+    <h2 class="home__downloads">
+      <p><%= number_with_delimiter(@downloads_count) %></p>
+      <span class="home__downloads__desc">downloads &amp; counting</span>
+    </h2>
+  <% end %>
```

Comment:
The button is centered now:

![rubygems org your community gem host 2015-07-01 12-04-02](https://cloud.githubusercontent.com/assets/118850/8459154/4e936b7c-1fe9-11e5-99bc-03442c982c8c.png)